contrib/executor/cypress/pkg/runner/cypress.go

```diff
@@ -32,24 +34,37 @@ func NewCypressRunner(dependency string) (*CypressRunner, error) {
 		dependency: dependency,
 	}
 
+	if params.ScrapperEnabled {
```

Comment:
Looks like this logic is repeated many times why not put it into lib?

Reply:
I am not sure how much value will it give extracting it in a separate function, it justs instantiates the logger using the factory method